Automattic/jetpack
# VideoPress: Cleanup old player references

#### Changes proposed in this Pull Request:
This PR removes references to the videopress_show_2015_player filter on WPCOM, which will become obsolete with D75134-code.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

This is a wpcom change, and there's no direct way to test this patch by itself (this is part of a global cleanup, see D75134-code stack).
One way could be to try some URLs both on JP and wpcom by adding /amp.
If I find a better way to test it "standalone", I'll update this PR.
